testing.py

```diff
@@ -1521,3 +1521,32 @@ def raise_for_status(self):
         implemented by real requests Response objects.
         """
         pass
+
+
+@pytest.fixture(autouse=True, scope="session")
```

Comment:
Move this back to testing.py, and use it as a pytest plugin, so it can be pulled into circulation.